free-2.md

```diff
@@ -34,36 +38,44 @@ _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+I imagine the program can be run on the terminal. At first, the user will be able to import a picture they already own, or input keywords to get a picture randomly from Google.
+The user might find it inconvenient that they can't see the final product of their coloring page on the terminal since this program is not interactive that way.
+
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy for the user to generate coloring pages with pictures they already own or pictures found on Google based on their input keywords. They will also be able to specify their own arbitrary settings such as removing all colors from the original pictures except purple.
+However, it would be impossible or very difficult for the user to take a peak on the final product before the program ends. In other words, they wouldn't be able to interact with the final product in real-time.
```

Comment:
Although you're suggesting real-time adjustments would not be possible, would the user be able to iteratively make adjustments to the output by running the program on previous outputs?